packages/runtime/test/consumption/notifications.test.ts

```diff
@@ -1,7 +1,7 @@
-import { ConsumptionIds, LocalNotificationStatus } from "@nmshd/consumption";
```

Comment:
Just a question out of interest, was it automatically or manually changed that `LocalNotificationStatus` is now imported from `"../../src"` (belongs to the runtime) instead of `"@nmshd/consumption"`?

Reply:
manually :(

Reply:
Oh dear! 😢

Reply:
But it's easy to find :) Just search for `from "@nmshd/consumtion"` in runtime tests (which should not be used) and bam.

Reply:
Was my approach too, I'll do it again as soon as you have removed the `wip` label 😝